pyvista/core/filters/image_data.py

```diff
@@ -1604,3 +1606,287 @@ def _update_and_get_output():
         # Restore active scalars
         self.set_active_scalars(scalars, preference='point')  # type: ignore[attr-defined]
         return output
+
+    def label_connectivity(
+        self,
+        *,
+        scalars: str | None = None,
+        scalar_range: str | VectorLike[int] | VectorLike[float] | None = 'auto',
+        extraction_mode: Literal['all', 'largest', 'point_seeds'] = 'all',
+        point_seeds: MatrixLike[float] | Sequence[float] | _vtk.vtkDataSet | None = None,
```

Comment:
Is `Sequence` meant to capture the single-vector case?
```suggestion
        point_seeds: MatrixLike[float] | VectorLike[float] | _vtk.vtkDataSet | None = None,
```